lib/Server.js

```diff
@@ -552,6 +552,21 @@ class Server {
           searchParams.set("password", webSocketURL.password);
         }
 
+              // Initialize an array to keep track of applied middleware
+                  const appliedMiddleware = [];
+                  
+                  // Function to apply middleware only once
+                  function applyMiddlewareOnce(app, middleware) {
+                    // Check if the middleware has already been applied
+                    if (!appliedMiddleware.includes(middleware)) {
+                      // Apply the middleware
+                      app.use(middleware);
+                      
+                      // Add the middleware to the applied middleware array
+                      appliedMiddleware.push(middleware);
+                    }
+                  }
```

Comment:
Why we need it here?

Reply:
@alexander-akait This code is used to ensure that middleware is applied only once to an Express.js application. Middleware functions are often applied to handle tasks like authentication, logging, or error handling. By keeping track of applied middleware in an array and checking if a middleware has already been applied before applying it again, this code prevents duplicate application of middleware, which can cause unexpected behavior or performance issues in an Express.js application.

Reply:
Please avoid extra code from PR and put only required for your improvement/fix

Reply:
Okay i will take care of it for further contribution